package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "percy-seleniumjs",
+  "version": "0.1.0-alpha.2",
+  "description": "Unofficial SeleniumJS SDK for Percy. Works with Protractor too.",
+  "main": "lib/index.js",
+  "scripts": {
+    "lint": "eslint ."
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ]
+  },
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "prettier": {
+    "printWidth": 79,
+    "singleQuote": true,
+    "trailingComma": "all"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/4Catalyzer/percy-seleniumjs.git"
+  },
+  "files": [
+    "lib"
+  ],
+  "author": "4Catalyzer",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/4Catalyzer/percy-seleniumjs/issues"
+  },
+  "homepage": "https://github.com/4Catalyzer/percy-seleniumjs#readme",
+  "peerDependencies": {
+    "selenium-webdriver": "^3.0.0 || ^4.0.0-alpha"
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.0.1",
+    "eslint": "^5.6.1",
+    "eslint-config-4catalyzer": "^0.4.4",
+    "eslint-config-prettier": "^3.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-prettier": "^3.0.0",
+    "husky": "^1.1.0",
+    "jest": "^23.6.0",
+    "lint-staged": "^7.3.0",
+    "prettier": "^1.14.3",
+    "rimraf": "^2.6.2"
+  },
+  "dependencies": {
+    "mime-types": "^2.1.21",
+    "percy-client": "^3.0.1",
+    "walk": "^2.3.14"
+  },
+  "release": {
+    "publishDir": "lib"
```

Comment:
nothing is using this

Reply:
that depends on the tool you're using. `npm` or `yarn` don't do anything with this by default.

if you're using our CI, then it will be respected

Reply:
you can see you're not publishing only lib: https://unpkg.com/percy-seleniumjs@0.1.0-alpha.3/